exonum/src/proto/schema/exonum/messages.proto

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The messages collection for the default Exonum consensus implementation.
+
+syntax = "proto3";
+
+package exonum.messages;
+
+option java_package = "com.exonum.core.messages";
+
+import "types.proto";
+import "runtime.proto";
+import "google/protobuf/timestamp.proto";
+
+// Container for the signed messages.
+message SignedMessage {
+  // Payload of the message as a serialized `ExonumMessage`.
+  bytes payload = 1;
+  // Public key of the author of the message.
+  exonum.crypto.PublicKey author = 2;
+  // Digital signature over the payload created with a secret key of the author of the message.
+  exonum.crypto.Signature signature = 3;
+}
+
+// Container for a verified message.
+message Verified {
+  SignedMessage raw = 1;
+}
+
+message CoreMessage {
+  oneof kind {
+    exonum.runtime.AnyTx any_tx = 1;
+    Precommit precommit = 4;
```

Comment:
It may be better to change the numbering so that the `Precommit` message will have id = 2 to make numbering continuous.

Reply:
...But this will break compatibility with previous Exonum versions. Are we OK with this?

Reply:
I think so. Since there are no migrations in the previous release, and there are a lot of breaking changes already, this won't cause any serious harm IMHO.